src/glue/meson.build

```diff
@@ -0,0 +1,14 @@
+module_droid_glue = shared_module('module-droid-glue',
+  ['module-droid-glue.c', '/usr/share/audioflingerglue/hybris.c'],
+  include_directories : [configinc],
+  c_args : [pa_c_args, '-I/usr/include/audioflingerglue', '-DPA_MODULE_NAME=module_droid_glue'],
```

Comment:
Hardcoding audioflingerglue paths in /usr/share isn't ideal here, it might be good to fix that to install a pkg-config file and a shared or even static library.